docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,340 @@
 {
```

Comment:
<p>I would add examples describing how/when (i.e., hypotetical scenarios that our experience tells us our users may be facing) the actions could/should be used. Basically, helping the users to 'see' how they would use those actions when writing their applications with RP.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>